### PR TITLE
[Web] Create components for Topbar and Toolbars

### PR DESCRIPTION
**Pull request by Sarfraz-droid**
Navbar Setup for #293 

This PR is parent to #302 

## Screenshots or GIF (In case of UI changes):
Peek 2022-06-24 18-16

## Types of changes

- [X] New feature (non-breaking change which adds functionality)

## Checklist:
- [X] My code follows the code style of this project.

### CODE REVIEW

File: src/helpers/appBar.js [hunk 1]
```diff
@@ -149,136 +151,147 @@ const CodeLabzAppBar = () => {
...
-              data-testId="navbarBrand"
-            >
-              <Link to={"/"}>
-                <BrandName />
-              </Link>
-            </Typography>
+        <MainNavbar />
+      </div>
+    );
+    // return (
```
Comment (Niweera) [inline src/helpers/appBar.js:157]: why is the lines are commented? any reason?

Comment (Niweera) [reply src/helpers/appBar.js:157]: Can you move it to another file so that our code keep clean. Leaving a large chunk of commented code is not nice noh.

Comment (Sarfraz-droid) [reply src/helpers/appBar.js:157]: Okay, Got it suree

Comment (Sarfraz-droid) [reply src/helpers/appBar.js]: Done!

**Review by Niweera — APPROVED** [review]

**Review by AsithaIndrajith — APPROVED** [review]

**Pull request merged**